dev-docs/backend.md

```diff
@@ -0,0 +1,54 @@
+# What is backend in inspec?
+
+Inspec::Backend class creates the train connection object. Train lets you talk to your local or remote operating systems and APIs with a unified interface.
+
+It allows you to:
+
+* execute commands via run_command
+* interact with files via file
+* identify the target operating system via os
+* authenticate to API-based services and treat them like a platform
+
+When inspec exec command is run with --target option it establishes the connection to that target using speficifed transport like ssh, winrm etc. By default it uses local transport if the --target is not provided. Backend class is responsible for creating this transprot and establish connection using train.
```

Comment:
You should probably mention that tran transports are plugins - gems that can be externally loaded and maintained outside of InSpec and Train.